edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeIdentitiesCache.cs

```diff
@@ -107,10 +89,14 @@ await serviceIdentity
             {
                 Events.ErrorRefreshingCache(e, id);
             }
+        }
 
-            if (invokeServiceIdentitiesUpdated)
+        public async Task RefreshServiceIdentities(IEnumerable<string> ids)
```

Comment:
>RefreshServiceIdentities [](start = 26, length = 24)

my neg-caching PR went in, I think you're gonna get merge conflicts here.  Lemme know if you need help resolving.